bee-common/bee-common-derive/tests/packable/fail/invalid_tag_enum.stderr

```diff
@@ -0,0 +1,5 @@
+error: All variants of an enum that derives `Packable` require a `#[packable(tag = ...)]` attribute.
```

Comment:
This error doesn't feel right ? All variants actually have a `#[packable(tag = ...)]` ?

Reply:
so the thing is that I'm parsing `#[packable(tag = <INTEGER>)]`. Maybe I should parse `#[packable(tag = <EXPR>)]` if the first one fails and return an error.